build.gradle.kts

```diff
@@ -1,18 +1,6 @@
-buildscript {
-  repositories {
-    mavenCentral()
-    gradlePluginPortal()
-  }
-
-  dependencies {
-    classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.8.21") // Don't forget to update KSP.
-    classpath("com.vanniktech:gradle-maven-publish-plugin:0.24.0")
-    classpath("org.jetbrains.dokka:dokka-gradle-plugin:1.8.10")
-  }
-}
-
-allprojects {
-  repositories {
-    mavenCentral()
-  }
+plugins {
+    //trick: for the same plugin versions in all sub-modules
```

Comment:
I don't fully understand what this comment is saying. Wanna remove it?

Reply:
Its there by default when starting with a Multiplatform template. Will remove it.